vassal-app/src/main/java/VASSAL/counters/Embellishment.java

```diff
@@ -1295,10 +1237,6 @@ else if (suffix.isSelected()) {
           isPrefix.set(index, NEITHER);
         }
       }
-      else if (index == 0) {
```

Comment:
Oh boy. I wonder if this was supposed to be an `index < 0` case.

Reply:
Hmm. This was an `else` until 9c43d480df8.

Reply:
@BrentEaston Do you have any memory of what you were trying to do in that commit? The code in this block won't have run since then, as the test a bit above is `index >= 0`.

Reply:
The offending commit comment says

Fixed bug 2442887: Exception if no Layer levels and type new level name

All I managed to do is stop the else from ever being executed, which was presumably the actual correct solution - remove the else altogether.  The actual problem being the if nothing has been selected yet, index will be -1 which will blow up the code in the else.